app/src/main/java/com/sunasterisk/musixmatch/ui/music/albumstab/albumdetails/AlbumDetailsPresenter.java

```diff
@@ -0,0 +1,53 @@
+package com.sunasterisk.musixmatch.ui.music.albumstab.albumdetails;
+
+import com.sunasterisk.musixmatch.data.model.Album;
+import com.sunasterisk.musixmatch.data.model.Track;
+import com.sunasterisk.musixmatch.data.repository.TrackRepository;
+import com.sunasterisk.musixmatch.data.source.Callback;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by superme198 on 05,April,2019
+ * in Music_Match__1.
+ */
+public class AlbumDetailsPresenter implements AlbumDetailsContract.Presenter {
+
+    private TrackRepository mRepository;
+    private AlbumDetailsContract.View mView;
+    private Album mAlbum;
+
+    public AlbumDetailsPresenter(TrackRepository repository, AlbumDetailsContract.View view, Album album) {
+        mRepository = repository;
+        mView = view;
+        mAlbum = album;
+    }
+
+    @Override
+    public void getTracksFollowAlbumID(List<Track> trackList, int id) {
```

Comment:
```suggestion
    public void getTracks(List<Track> trackList, int id) {
```
e nên sử dụng tính chất đa hình
getTracks(int albumId, callback)
method này cần viết trong datasource.
mà tại sao k query trong dabase ra mà lại lấy hết trong database ra rồi lại mất công for tiếp để tìm